synth.py

```diff
@@ -29,4 +29,6 @@
 templates = common_templates.node_library(
     source_location='build/src', versions=['v1'], default_version='v1')
 s.copy(templates, excludes=[])
+s.replace('src/v1/cloud_functions_service_client.ts', "function:string", "function_: string")
+s.replace('src/v1/cloud_functions_service_client.ts', "function: function", "function: function_")
```

Comment:
Do we plan to have this in perpetuity or is this something the generator should be able to handle? For any substantive synthtool replacements, we should have a bug filed on the upstream generator or team (and comment here with the link) so that these clients can generate without synthtool hacks.

Reply:
Yes! Alex F. is working on it. We discussed it in our Node meeting yesterday. He asked me to implement this hack in the meantime, as it might take some weeks to work.

Reply:
https://github.com/googleapis/gapic-generator-typescript/issues/609